sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -560,7 +560,7 @@ private void onFirstFrameDrawn(final @Nullable ISpan ttfdSpan, final @Nullable I
       if (ttfdSpan != null && ttfdSpan.isFinished()) {
         ttfdSpan.updateEndDate(endDate);
         // If the ttfd span was finished before the first frame we adjust the measurement, too
-        ttidSpan.setMeasurement(
+        ttfdSpan.setMeasurement(
```

Comment:
well spotted!

Reply:
I had to revert it, as it's not a simple enough change (the ttfd span is already finished and we can't set measurements). [Created an issue for this](https://github.com/getsentry/sentry-java/issues/3384)